basket/news/tasks.py

```diff
@@ -212,6 +212,24 @@ def fxa_source_url(metrics):
     return source_url
 
 
+@et_task
+def fxa_email_changed(data):
+    ts = data['ts']
```

Comment:
q: Is `data['ts']` type `float`?

Reply:
Yes, it should be a float in seconds; generated here for reference:

  https://github.com/mozilla/fxa-auth-server/blob/d205c9abee380e250413570b9873561f55bb5068/lib/log.js#L128